### PR TITLE
Update to v10

### PR DESCRIPTION
**Pull request by Boefjim**
# Changes
- Updated reference to character sheet
- Migrated manifest to v10 structure

Fixes: #63

### CODE REVIEW

**Comment by akrigline** [comment]
Thanks for this contribution, I see it but don't at the moment have time to review and merge it. I'm hopeful to be able to do so this week, or at the least in the next few weeks.

File: src/module.json [hunk 1]
```diff
@@ -1,26 +1,40 @@
...
 	"name": "compact-beyond-5e-sheet",
 	"title": "Compact DnDBeyond 5e Character Sheet",
 	"description": "A compact 5e character sheet with similar layout and functionality to dndbeyond's.",
-	"systems": ["dnd5e"],
 	"version": "0.20.0",
 	"author": "Andrew Krigline (akrigline)",
 	"authors": [
 		{
-			"name": "Andrew Krigline",
+			"name": "Andrew Krigline (akragline)",
```
Comment (akrigline) [inline src/module.json:11]:
```suggestion
			"name": "Andrew Krigline (akrigline)",
```

**Review by akrigline — APPROVED** [review]
Thanks for this! Have you noticed any compatibility warnings or the like on v10?

**Comment by Boefjim** [comment]
> Thanks for this! Have you noticed any compatibility warnings or the like on v10?

No problem!
Am happy to help, me and my friends have been using this character sheet for a while for our D&D sessions, and since we updated to Foundry V10 recently we really missed it, so I figured I might drop by and see what I can do to make it work again :P

There are definitely some deprecation warnings, most referring to the changes in the data models.
The other things are that the handlebars helper content option is also deprecated, and isObjectEmpty() should be replaced with isEmpty()

Haven't found any other problems so far, if you want I could create another PR to fix (most of) those deprecations ^w^

Also not sure how you've setup the build pipeline of the module, is it just manually adjusting the manifest file?
Because the new version doesn't seem to be available for updating yet

**Comment by akrigline** [comment]
Some more deprecation fixes in another PR would rock! I'm happy to do a hard cut between 0.20 and the next version to make it only v10 compatible.

I'll need to manually make a new release for it to become available for real, intend to do that tonight.

**Comment by Boefjim** [comment]
Aight, made a PR for it: #66
Couldn't figure out how to fix the handlebars helper content option deprecation, but that is something that can be removed later since that will only be actually removed in V12 :P

**Pull request merged**